src/portal/src/app/project/tag-retention/add-rule/add-rule.component.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
```

Comment:
All the ts file need to add license now? Why the license is 2017?

Reply:
modified